lesson/src/components/Home.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import Movie from './Movie'
+import './Home.css'
+
+class Home extends Component {
+
+    constructor(){
+        super()
+        this.state ={
+            searchVal:'',
+            searched:[]
+        }
+    }
+
+    searchupdate = (e) => {
+        this.setState({
```

Comment:
can also write it like
`searchUpdate = ({target}) => this.setState({searchVal: target.value})`